probatus/feature_elimination/feature_elimination.py

```diff
@@ -430,14 +465,14 @@ def fit(self, X, y):
                                          train_metric_std = np.round(np.std(scores_train), 3),
                                          val_metric_mean = np.round(np.mean(scores_val), 3),
                                          val_metric_std = np.round(np.std(scores_val), 3))
-
-            print(f'Round: {round_number}, Current number of features: {len(current_features_set)}, '
-                  f'Current performance: Train {self.report_df.loc[round_number]["train_metric_mean"]} '
-                  f'+/- {self.report_df.loc[round_number]["train_metric_std"]}, CV Validation '
-                  f'{self.report_df.loc[round_number]["val_metric_mean"]} '
-                  f'+/- {self.report_df.loc[round_number]["val_metric_std"]}. \n'
-                  f'Num of features left: {len(remaining_features)}. '
-                  f'Removed features at the end of the round: {features_to_remove}')
+            if self.verbose > 50:
+                print(f'Round: {round_number}, Current number of features: {len(current_features_set)}, '
+                      f'Current performance: Train {self.report_df.loc[round_number]["train_metric_mean"]} '
+                      f'+/- {self.report_df.loc[round_number]["train_metric_std"]}, CV Validation '
+                      f'{self.report_df.loc[round_number]["val_metric_mean"]} '
+                      f'+/- {self.report_df.loc[round_number]["val_metric_std"]}. \n'
+                      f'Num of features left: {len(remaining_features)}. '
+                      f'Removed features at the end of the round: {features_to_remove}')
         self.fitted = True
```

Comment:
just noticed this. Remember if you have attributes with a trailing _ in the class, sklearn sees it as a fitted attribute. See this source code https://github.com/scikit-learn/scikit-learn/blob/0fb307bf3/sklearn/utils/validation.py#L955

Reply:
#24 will cover this